docs/user_best_practices.md

```diff
@@ -0,0 +1,251 @@
+# Scarf User Guide & Best Practices
+
+## Intro
+
+This is a user guide to best practices when using Scarf.  Consider this not only a guide but also an FAQ.  
+
+Currently this guide contains:
+
+1. How to use Scarf Data in existing  sales and marketing efforts
+	1. Events you should be tracking
+	2. The goal is ongoing usage 
+	3. How to use Scarf to infer ongoing usage 
+	4. Basic lead scoring with Scarf
+	5. Sample customer journey 
+	6. What to do with companies identified by Scarf
+	7. Common ways to inject Scarf into existing sales/marketing activities 
+	8. Common ways for startups to build their first sales and marketing processes 
+2. How to use confidence intervals
+3. Best Practices to track multiple projects and packages
+	1. Handling different projects
+	2. Handling different versions
+	3. Handling different platforms
+	4. Container best practices
+	5. Variables 
+    6. Pixels for each project	
+4. Tracking external link clicks
+
+## How to use Scarf data in existing sales and marketing efforts
+
+### Events we should be tracking:
+
+There are 3 events I would suggest everyone track.  

```

Comment:
Instead of "I", should likely be "we" since we're speaking from the perspective of the Scarf organization here.

```suggestion
There are 3 events we would suggest everyone track. 
```